cleverpush/src/main/java/com/cleverpush/banner/AppBannerPopup.java

```diff
@@ -218,12 +220,27 @@ public void setOpenedListener(AppBannerOpenedListener openedListener) {
   }
 
   private View createLayout(int layoutId) {
+    boolean isExceptionGenerate = false;
+    View layout = null;
     try {
-      View layout = activity.getLayoutInflater().inflate(layoutId, null);
-      layout.setOnClickListener(view -> dismiss());
-      return layout;
+      layout = activity.getLayoutInflater().inflate(layoutId, null);
     } catch (Exception exception) {
       Logger.e(TAG, exception.getLocalizedMessage());
+      isExceptionGenerate = true;
+    } finally {
+      try {
+        if (isExceptionGenerate) {
```

Comment:
all the stuff in this if condition can be moved to above catch and then you dont need isExceptionGenerate anymore